### PR TITLE
Add JSON-schema

### PR DESCRIPTION
**Pull request by williamhogman**

### CODE REVIEW

File: schema-docs/PrivateUnsecuredLoanDelayedProcessing.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+* is optional
+* type: `enum`
+* defined in this schema
+
+The value of this property **must** be equal to one of the [known values below](#delayreason-known-values).
+
+### delayReason Known Values
+| Value | Description |
+|-------|-------------|
+| `HOLIDAY` | The application will be processed later because of a holiday |
```
Comment (zinob) [inline schema-docs/PrivateUnsecuredLoanDelayedProcessing.md]: perhaps we should write "bank holiday"

File: schema-docs/PrivateUnsecuredLoanDelayedProcessing.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+| `MANUAL_PROCESSING` | The application is being processed manually |
+| `OPERATIONAL_ISSUES` | The service provider is experiencing operational issues and processing will be delayed. |
+
```
Comment (zinob) [inline schema-docs/PrivateUnsecuredLoanDelayedProcessing.md]: Add an "other" (or OTTER | Servers have been eaten by adorable otters| )

File: schema/se/PrivateUnsecuredLoanApplicationCreated.yaml [hunk 3]
```diff
@@ -0,0 +1,380 @@
...
+  broker:
+    type: "string"
+    title: 'Domain-name of the broker, in reverse order'
+    description: "Reversed DNS name for the broker, for example, example.com becomes com.example"
+    pattern: "^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9-][a-zA-Z0-9]).)([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9-]*[A-Za-z0-9])"
+
+  brokerReference:
+    type: string
+    title: "Reference number for the broker"
+    description: "An arbitrary ID specified by the broker"
```
Comment (zinob) [inline schema/se/PrivateUnsecuredLoanApplicationCreated.yaml]: Should we mention something about that it must be unique per application? and might not be reused?

File: schema-docs/PrivateUnsecuredLoanOffering.md [hunk 4]
```diff
@@ -0,0 +1,753 @@
...
+| Can be instantiated | Yes | Experimental | No | Forbidden | Forbidden | [PrivateUnsecuredLoanOffering.json](PrivateUnsecuredLoanOffering.json) |
+
+# PrivateUnsecuredLoanOffering Properties
+
+| Property | Type | Required | Defined by |
+|----------|------|----------|------------|
+| [broker](#broker) | `string` | **Required** | PrivateUnsecuredLoanOffering (this schema) |
+| [brokerReference](#brokerreference) | `string` | **Required** | PrivateUnsecuredLoanOffering (this schema) |
+| [offer](#offer) | `object` | **Required** | PrivateUnsecuredLoanOffering (this schema) |
+
```
[the remark this answers is not in the document]
Comment (williamhogman) [reply schema-docs/PrivateUnsecuredLoanOffering.md]: There wasn't one in the previous version

[the remark this answers is not in the document]
Comment (williamhogman) [reply schema-docs/PrivateUnsecuredLoanOffering.md]: #39

File: schema-docs/PrivateUnsecuredLoanDelayedProcessing.md (hunk 2 above)
Comment (williamhogman) [reply schema-docs/PrivateUnsecuredLoanDelayedProcessing.md]: Will add other this was in the spec

File: schema-docs/PrivateUnsecuredLoanDelayedProcessing.md (hunk 1 above)
Comment (williamhogman) [reply schema-docs/PrivateUnsecuredLoanDelayedProcessing.md]: #40

File: schema/se/PrivateUnsecuredLoanApplicationCreated.yaml (hunk 3 above)
Comment (williamhogman) [reply schema/se/PrivateUnsecuredLoanApplicationCreated.yaml]: Yes

File: schema-docs/PrivateUnsecuredLoanDelayedProcessing.md (hunk 2 above)
Comment (williamhogman) [reply schema-docs/PrivateUnsecuredLoanDelayedProcessing.md]: Done

**Pull request merged**